driverkit/config/7.3.0+driver/x86_64/ubuntu-generic_5.15.0-124-generic_134~20.01.1.yaml

```diff
@@ -0,0 +1,7 @@
+kernelversion: 134~20.01.1
```

Comment:
Isn't this `134~20.04.1` ?
I found this one: https://ubuntu.pkgs.org/20.04/ubuntu-updates-main-amd64/linux-image-5.15.0-130-generic_5.15.0-130.140~20.04.1_amd64.deb.html